cli/list.go

```diff
@@ -17,8 +17,8 @@ package cli
 import (
 	"fmt"
 
-	log "github.com/DataDrake/waterlog"
 	"github.com/getsolus/usysconf/config"
+	"golang.org/x/exp/slog"
```

Comment:
```suggestion
	"log/slog"
```

Idem for all other occurrences.

Reply:
This is pure personal convenience. Fedora so far ships Go 1.20 which hasn't slog in its standard library. I think it's harmless to keep it like that for a while.